js/actions/DeploymentActions.js

```diff
@@ -0,0 +1,59 @@
+var oboe = require("oboe");
+
+var config = require("../config/config");
+var AppDispatcher = require("../AppDispatcher");
+var DeploymentEvents = require("../events/DeploymentEvents");
+
+var DeploymentActions = {
+  requestDeployments: function () {
+    this.request({
+      url: config.apiURL + "v2/deployments"
+    }).done(function (deployments) {
```

Comment:
I'd like for the `.done(function (deployments) {` to be on the next line, but that's a style preference